content/tutorials/kit-assembly.md

```diff
@@ -39,22 +46,23 @@ Make sure to read all these **before** you start assembly
 3. Connect the Power Board to the Pi via one of the black micro-usb cables; the standard USB end goes into any USB socket on the Pi or connected USB hub, the micro-usb end into the Power Board.
-4. Connect the Motor Board to the Power Board by screwing the two large (7.5mm) camcons provided onto the opposite ends of a pair of wires, ensuring that positive connects to positive and ground to ground, and then plugging one end into the appropriate socket of the Motor Board and the other into a high power socket on the side of the Power Board.
+4. Connect the Motor Board to the Power Board by screwing the two large (7.5mm) camcons provided onto the opposite ends of a pair of wires, ensuring that positive connects to positive and ground to ground, and then plugging one end into the appropriate socket of the Motor Board (the connections labelled M0 or M1) and the other into a high power socket on the side of the Power Board ().
 5. Connect the Motor Board to the Pi by way of the other black micro-usb cable; the standard USB end goes into any USB socket on the Pi or connected USB hub, the micro-usb end goes into the Motor Board.
 6. The Arduino Servo Shield should already be connected into the top of the Arduino, but, if not, the pins on the bottom fit directly into the sockets with the same labels on the top of the Arduino.
-7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
+7. Connect the Arduino to the Pi by way of the beige USB-A to USB-B cable; the square-ish USB-B (this is the same cable the connects to a printer) end goes into the appropriate metal-cased connector on the Arduino, the standard USB end goes into any free USB port on the Pi or connected USB hub. This will provide both power and communication to the Arduino.
 8. Connect the Arduino Servo Shield to the Power Board by screwing the green connector on the top and the small (3.81mm) camcon provided onto the opposite ends of a pair of wires and plugging the camcon into the second of the 5V power rail sockets on the Power Board.
 9. Connect the USB Webcam to the Pi via any one of the remaining USB sockets on the Pi or connected USB hub.
 10. To connect the motors, first screw wires connected to their pins into one of the medium-sized (5mm) camcons and plug these into either of the two appropriate sockets on a Motor Board.
-11. At this point, check that everything is connected up correctly and ask someone in a purple Smallpeice T-shirt to double-check then we will give you your battery. You don’t want the battery to blow up your kit!
+11. At this point, check that everything is connected up correctly (it may be helpful to ask someone around you to check that all cables are connected properly, ground to ground, positive to positive).
```

Comment:
Probably dont need the explicit _"ground to ground, positive to positive"_ here.